recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -23,91 +26,84 @@ class CubicInterpolationConan(ConanFile):
         "fPIC": True,
     }
 
-    generators = "cmake", "cmake_find_package"
     _cmake = None
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _is_msvc(self):
```

Comment:
This should be replace with the help `is_msvc` :) https://docs.conan.io/2/reference/tools/microsoft/helpers.html?highlight=is_msvc#conan.tools.microsoft.visual.is_msvc